app/controllers/batches_controller.rb

```diff
@@ -116,8 +116,10 @@ def create
 
   def pipeline
     # All pipeline batches routes should just direct to batches#index with pipeline and state as filter parameters
-    @batches = Batch.where(pipeline_id: params[:pipeline_id] || params[:id]).order(id: :desc).includes(:user,
-                                                                                                       :pipeline).page(params[:page])
+    @batches = Batch.where(pipeline_id: params[:pipeline_id] || params[:id])
+                    .order(id: :desc)
+                    .includes(:user, :pipeline)
+                    .page(params[:page])
   end
```

Comment:
Just some line length changes while I worked out what I needed to do here.
As it was, nothing